frontend/src/containers/issue/components/issueUpperBox.js

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+import React, { Component } from 'react';
+import renderLabels from './label'
+
+const IssueUpperBox = styled.div`
+        a {
+            text-decoration:none;
+            color: black;
+        }
+        a:hover {
+            text-decoration:none;
+            color: blue;
+        }
+
+    `;
+
+const renderIssueUpperBox = (data) => {
+    const href = `/issues/${data.issue.id}`
+    return (<IssueUpperBox>
+        <a href={href}>{data.issue.title}</a>
+        {renderLabels(data.issue.issue_labels)}
+    </IssueUpperBox>);
+}
```

Comment:
```suggestion
const renderIssueUpperBox = ({issue}) => {
    const href = `/issues/${issue.id}`
    return (<IssueUpperBox>
        <a href={href}>{issue.title}</a>
        {renderLabels(issue.issue_labels)}
    </IssueUpperBox>);
}
```
이렇게 하면 data.issue에 여러번 접근하는 것을 줄일 수 있을 것 같습니다!